polaris-react/src/components/TextField/TextField.tsx

```diff
@@ -481,7 +500,7 @@ export function TextField({
           onClick={handleClick}
         >
           {prefixMarkup}
-          {input}
+          {verticalContent ? inputWithVerticalContentMarkup : input}
```

Comment:
Let's move the conditional out of the return by making it a variable.

```suggestion
const inputMarkup = verticalContent ? inputWithVerticalContentMarkup : input;
...
          {inputMarkup}
```